oidc-service-impl/src/test/java/io/mosip/esignet/services/AuthorizationServiceTest.java

```diff
@@ -1177,6 +1189,78 @@ public void getConsentDetails_withInvalidTransaction_thenFail(){
         }
     }
 
+    @Test
+    public void testSendOtp_ValidRequest_thenPass() throws Exception {
+        OtpRequest otpRequest = new OtpRequest();
+        otpRequest.setCaptchaToken("captchaToken");
+        otpRequest.setTransactionId("transactionId");
+        otpRequest.setIndividualId("individualId");
+        ArrayList<String> otpChannels=new ArrayList<>();
+        otpRequest.setOtpChannels(otpChannels);
+
+        OIDCTransaction transaction = new OIDCTransaction();
+        transaction.setTransactionId("transactionId");
+        transaction.setIndividualIdHash("individualIdHash");
+        transaction.setRelyingPartyId("relyingPartyId");
+        transaction.setClientId("clientId");
+        transaction.setAuthTransactionId("transactionId");
+        when(cacheUtilService.getPreAuthTransaction(Mockito.anyString())).thenReturn(transaction);
+        when(cacheUtilService.updateIndividualIdHashInPreAuthCache(Mockito.anyString(), Mockito.anyString())).thenReturn(transaction);
+        when(cacheUtilService.isIndividualIdBlocked(Mockito.anyString())).thenReturn(false);
+
+        SendOtpResult sendOtpResult = new SendOtpResult();
+        sendOtpResult.setTransactionId("transactionId");
+        sendOtpResult.setMaskedEmail("maskedEmail");
+        sendOtpResult.setMaskedMobile("maskedMobile");
+
+        SendOtpDto sendOtpDto=new SendOtpDto();
+        sendOtpDto.setTransactionId("transactionId");
+        sendOtpDto.setIndividualId("individualId");
+        ArrayList<String> otpChannel=new ArrayList<>();
+        sendOtpDto.setOtpChannels(otpChannel);
+
+        Mockito.when(authenticationWrapper.sendOtp("relyingPartyId","clientId",sendOtpDto)).thenReturn(sendOtpResult);
+        OtpResponse otpResponse = authorizationServiceImpl.sendOtp(otpRequest);
+        Assert.assertNotNull(otpResponse);
+        Assert.assertEquals("transactionId", otpResponse.getTransactionId());
+        Assert.assertEquals("maskedEmail", otpResponse.getMaskedEmail());
+        Assert.assertEquals("maskedMobile", otpResponse.getMaskedMobile());
+    }
+
+    @Test
+    public void testSendOtp_InValidRequest_thenFail() throws Exception {
```

Comment:
Can be specific with the test case naming, its fails because of blocked individual Id.
eg: sendOtp_whenIndividualIdBlocked_thenFail

Reply:
ok @ase-101